common/containers/Tabs/SendTransaction/index.tsx

```diff
@@ -265,6 +271,8 @@ export class SendTransaction extends React.Component<Props, State> {
         ? getDecimal('ether')
         : (this.state.token && this.state.token.decimal) || 0;
     const isWeb3Wallet = this.props.wallet instanceof Web3Wallet;
+    const isLedgerWallet = this.props.wallet instanceof LedgerWallet;
+    const isTrezorWallet = this.props.wallet instanceof TrezorWallet;
```

Comment:
a utility function could be helpful as additional hardware wallets become supported, but its not worth making holding up the PR.